main.py

```diff
@@ -1,11 +1,18 @@
 import argparse
 import sys
-from stix_shifter import stix_shifter
+from stix_shifter.stix_translation import stix_translation
+from stix_shifter.stix_transmission import stix_transmission
 import json
 
 
 TRANSLATE = 'translate'
 TRANSMIT = 'transmit'
+RESULTS = 'results'
+QUERY = 'query'
+DELETE = 'delete'
+STATUS = 'status'
+PING = 'ping'
+IS_ASYNC = 'is_async'
 
```

Comment:
these same variables are declared in three places. could we keep those in one place may be in main or inside stix translation/transmission if thats easier?